### DISCUSSION TITLE
Invite by SMS

### DISCUSSION THREAD

**Discussion by xurshid29** [supabase/supabase#3979] [Questions] [answered]
Hi, all! How to invite a user by SMS?

**Comment by J0** [comment, marked as answer]
Hey,

We support sign in/up via SMS but I don't think we support invitations via SMS at the moment. One way this could be implemented is to make use of the `generateLink` endpoint to get an invite link and then send that out via SMS.

Hope this helps in some way.
Thanks!

Joel
